cypress.e2e.config.ts

```diff
@@ -117,5 +125,6 @@ export default defineConfig({
         },
         baseUrl: 'http://localhost:8000',
         specPattern: 'cypress/e2e/**/*.{js,jsx,ts,tsx}',
+        chromeWebSecurity: false,
```

Comment:
gamely attempting to get CI to pass, followed what Georgiy did [here](https://github.com/PostHog/posthog/pull/25458)